src/sidebar/util/export-annotations.ts

```diff
@@ -0,0 +1,27 @@
+import type { Group } from '../../types/api';
+
+const formatDate = (date: Date): string => {
+  const year = date.getFullYear();
+  const month = `${date.getMonth() + 1}`.padStart(2, '0');
+  const day = `${date.getDate()}`.padStart(2, '0');
+
+  return `${year}-${month}-${day}`;
+};
+
+type SuggestedFilenameOptions = {
+  group: Group | null;
```

Comment:
Can we have this function take group name as a string so it doesn't have to be aware of the `Group` type? Otherwise it's more of a helper than a util.